api/draft_registrations/serializers.py

```diff
@@ -20,13 +20,18 @@
 )
 from api.taxonomies.serializers import TaxonomizableSerializerMixin
 from osf.exceptions import DraftRegistrationStateError
+from osf.models import Node
 from website import settings
 
 
 class NodeRelationshipField(RelationshipField):
 
     def to_internal_value(self, node_id):
-        node = self.context['view'].get_node(node_id=node_id) if node_id else None
+        context_view = self.context['view']
+        if hasattr(context_view, 'get_node'):
+            node = self.context['view'].get_node(node_id=node_id) if node_id else None
+        else:
+            node = Node.load(node_id)
```

Comment:
After the upgrade, one test fails due to `self.context['view']` returning an object not having `.get_node`. There is one concern of this fix since `.get_node()` checks permission and other stuff before loading the node. Might worth another look at the failed test.